### PR TITLE
RB5: update firmware-qcom-rb5 and linux-linaro-qcomlt

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

File: recipes-bsp/firmware/firmware-qcom-rb5_20210331-v4.bb [hunk 1]
```diff
@@ -12,8 +12,13 @@ LIC_FILES_CHKSUM = "file://LICENSE.qcom.txt;md5=cbbe399f2c983ad51768f4561587f000
...
 "
-SRC_URI[md5sum] = "d9289f59fe4f93ce433707294c9286ca"
-SRC_URI[sha256sum] = "9d7b42916d83c8f721258175b8d7a9ed758ebe02228d36099e6ea1a2b2a556d3"
+SRC_URI[md5sum] = "d65ec09ba18dcafe291c870e0516c290"
+SRC_URI[sha256sum] = "30e2c02be32de9f809b590f4fe76d9eb66d35f8c7d13b1f2850beb3d793192cc"
+
+# From v2 to v4 the versioning has changed, so add epoch
+# 20210118133815-v2
+# 20210331-v4
+PE = "1"
```
Comment (ndechesne) [inline recipes-bsp/firmware/firmware-qcom-rb5_20210331-v4.bb:21]: nice! It's not often you get a chance to use PE!

File: recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2014-2020 Linaro
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb:1]: we are in 2021 now!

File: recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+DESCRIPTION = "Linaro Qualcomm Landing team 5.12 Kernel"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+
+require recipes-kernel/linux/linux-linaro-qcom.inc
+require recipes-kernel/linux/linux-qcom-bootimg.inc
+
+LOCALVERSION ?= "-linaro-lt-qcom"
+
+SRCBRANCH = "release/rb5/qcomlt-5.12"
```
Comment (ndechesne) [inline recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb:12]: this commit should remove the 5.11 recipe, no? since it won't be used anymore, no?

Comment (lumag) [reply recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb:12]: Done

File: recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb (hunk 2 above)
Comment (lumag) [reply recipes-kernel/linux/linux-linaro-qcomlt_5.12.bb]: Done

**Comment by lumag** [comment]
@ndec: updated both PRs

**Pull request merged**